clients/src/main/java/org/apache/kafka/common/requests/FetchRequest.java

```diff
@@ -139,9 +148,36 @@
      */
     private static final Schema FETCH_REQUEST_V6 = FETCH_REQUEST_V5;
 
+    // FETCH_REQUEST_V7 added incremental fetch requests.
+    public static final Field.Int32 SESSION_ID = new Field.Int32("session_id", "The fetch session ID");
+    public static final Field.Int32 EPOCH = new Field.Int32("epoch", "The fetch epoch");
+
+    private static final Schema FORGOTTEN_TOPIC_DATA = new Schema(
+        TOPIC_NAME,
+        new Field(PARTITIONS_KEY_NAME, new ArrayOf(Type.INT32),
+            "Partitions to remove from the fetch session."));
+
+    private static final Schema FETCH_REQUEST_V7 = new Schema(
+        new Field(REPLICA_ID_KEY_NAME, INT32, "Broker id of the follower. For normal consumers, use -1."),
+        new Field(MAX_WAIT_KEY_NAME, INT32, "Maximum time in ms to wait for the response."),
+        new Field(MIN_BYTES_KEY_NAME, INT32, "Minimum bytes to accumulate in the response."),
+        new Field(MAX_BYTES_KEY_NAME, INT32, "Maximum bytes to accumulate in the response. Note that this is not an absolute maximum, " +
+            "if the first message in the first non-empty partition of the fetch is larger than this " +
+            "value, the message will still be returned to ensure that progress can be made."),
+        new Field(ISOLATION_LEVEL_KEY_NAME, INT8, "This setting controls the visibility of transactional records. Using READ_UNCOMMITTED " +
```

Comment:
nit: These are the same comments as above. How about creating a static `Field` instances or at least extracting the message.

Reply:
max_bytes does have different doc strings in different message versions, though.  I started looking at adding more constants for this, but it got a bit messy-- maybe a good follow on change?